### DISCUSSION TITLE
Can we use inline comments within slice files and expect inline comments in generated C# code?

### DISCUSSION THREAD

**Discussion by kamyar1979** [icerpc/icerpc-csharp#3947] [Q&A]
Some IDL languages has the capability of inline comments, and the generator copies the comments in place of the generated function code. Can we have the same feature here?

**Comment by pepone** [comment]
Hi,

With Slice only the doc comments are copied to the generated code, see https://docs.icerpc.dev/slice2/language-guide/comments
